packages/grid/x-data-grid/src/hooks/features/rows/useGridParamsApi.ts

```diff
@@ -113,6 +104,7 @@ export function useGridParamsApi(apiRef: React.MutableRefObject<GridPrivateApiCo
         tabIndex: cellTabIndex && cellTabIndex.field === field && cellTabIndex.id === id ? 0 : -1,
         value,
         formattedValue: value,
+        isEditable: false,
```

Comment:
Why is this necessary? It's overridden a few lines down

Reply:
JS engines can create the object shape/map/hidden-class in one go, without transition.